cscs-checks/mch/automatic_arrays.py

```diff
@@ -4,62 +4,60 @@
 
 @rfm.simple_test
 class AutomaticArraysCheck(rfm.RegressionTest):
-    def __init__(self, **kwargs):
+    def __init__(self):
         super().__init__()
         self.valid_systems = ['daint:gpu', 'dom:gpu', 'kesch:cn']
-        self.valid_prog_environs = ['PrgEnv-cray*', 'PrgEnv-pgi*',
```

Comment:
Shouldn't this test be supported by additional Cray and PGI prgenvs?

Reply:
The code uses MPI behind the scenes, although it runs with a single task.